tekton/resources/images/docker-multi-arch-template.yaml

```
@@ -89,8 +89,8 @@ spec:
             for tag in ${TAGS}; do DESTINATIONS="${DESTINATIONS} --tag ${IMAGE_URL}:${tag}"; done
             IFS=$OIFS
 
-            gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
-            gcloud auth configure-docker
+            IMAGE_REGISTRY_USER=tekton-robot 
+            docker login ghcr.io -u ${IMAGE_REGISTRY_USER} --password-stdin < ${"$IMAGE_REGISTRY_CREDENTIALS"}
```

This causes an error due to unterminated string. The following works:

```suggestion
            docker login ghcr.io -u ${IMAGE_REGISTRY_USER} --password-stdin < "${IMAGE_REGISTRY_CREDENTIALS}"
```